src/components/CardGallery.jsx

```diff
@@ -0,0 +1,18 @@
+import "../styles/Card.scss";
+
+export default function Gallery({ cards }) {
```

Comment:
Even though this seems like a completely new file, it's just a rename for `Card.jsx`. GitHub sometimes doesn't acknowledge these changes properly.